lib/game.js

```diff
@@ -171,5 +173,15 @@ Game.prototype.instantiatePrefab = function(name) {
 	}.bind(this));
 	return id;
 };
-
-module.exports = Game;
+Game.prototype.registerPrefab = function (name,obj){
+	var prefabs = this.prefabs;
+	prefabs[name] = obj;
+};
+Game.prototype.importPrefabs = function(library){
```

Comment:
Do you mean hide `registerPrefab`? No, I think both of them are useful. If you think there should only be one method, I'd get rid of `importPrefabs`/`registerPrefabs` and let the user do the loop.